### PR TITLE
feat(other): development

### LINKED ISSUE #26: 🚀 [Feature] Write content for the Homepage
## 🚀 Feature

Describes the following points:
- Benefits, 
- main features, 
- demo (link) 
- get started (link), 
- video (link) 
- testimonials?, 
- donation, 
- contact (link)

### LINKED ISSUE #27: 🚀 [Feature] Write content for Feature Page
## 🚀 Feature

Write some content for the page Feature:
- see Wiki (user guide), 
- screenshots

### LINKED ISSUE #28: 🚀 [Feature] Content for "Get started" Page
## 🚀 Feature

Wirte content for "Get started" page:
- Prerequisites
- Installation guide

### LINKED ISSUE #30: 🚀 [Feature] Content for Contribute Page
## 🚀 Feature

Write content for Contribute Page:

- donation possibilities
  - for inspiratons see https://soutenir.framasoft.org/en/
- Discord, 
- translate, 
- development, 
- marketing, 
- etc. →see board

### LINKED ISSUE #32: 🚀 [Feature] Content for Ressources Page
## 🚀 Feature

Write content for Ressources Page:
- Documentation, 
- installation guide, 
- user guide (wiki), 
- FAQs, 
- GitHub (link)

### LINKED ISSUE #34: 🚀 [Feature] Content for Contact Page
## 🚀 Feature

Write Content for Contact Page:
- Address, 
- e-mail, 
- GitHub, 
- Discord, 
- operator meetings, 
- Matrix room

### LINKED ISSUE #35: 🚀 [Feature] Content for Privacy Policy Page
## 🚀 Feature

Write content for Privacy Policy Page

### LINKED ISSUE #36: 🚀 [Feature] Content for Impressum Page
## 🚀 Feature

Write content for Impressum Page

### LINKED ISSUE #47: 🚀 [Feature] Content for Demo Page
## 🚀 Feature

Content for Demo Page.

### Todo

- presenting the demo networks

### LINKED ISSUE #54: 🚀 [Feature] Content for Features Page
## 🚀 Feature

Content for Features Page.

### Todo

- [ ] add the page

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Development

### Issues

- relates #49
- fixes #26 
- fixes #27 
- fixes #28 
- fixes #30 
- fixes #32 
- fixes #34 
- fixes #35 
- fixes #36 
- fixes #47 
- fixes #54 

### Todo

- [x] hero full width?
- [x] define SCSS variables for colors
- [x] add privacy policy #35 
- [x] clean up:
  - [x] search `Wolle`

### CODE REVIEW

File: docs/de/news/README.md
Comment (Tirokk) [inline docs/de/news/README.md:1]: Check this before merging into master!

File: docs/en/README.md
Comment (Tirokk) [inline docs/en/README.md:1]: Check this before merging into master!

File: docs/.vuepress/config/theme.js [hunk 1]
```diff
@@ -23,72 +23,96 @@ export default hopeTheme({
...
+        },
+        {
+          text: 'Mitmachen',
+          link: '/de/contribute/'
         },
         {
-          text: 'Demo',
-          link: 'https://stage.ocelot.social/',
+          text: 'Dokumentation',
+          link: 'https://docs.ocelot.social/' ,
```
Comment (Elweyn) [inline docs/.vuepress/config/theme.js:43]:
```suggestion
          link: 'https://docs.ocelot.social/',
```

File: docs/de/contribute/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+Die Software des Netzwerks ist frei verfügbar – Betrieb, Unterhalt und Weiterentwicklung kosten hingegen Geld.
+
+Der Verein [busFaktor() e.V.](https://busfaktor.org/de/) koordiniert und fördert die Programmierung von *ocelot.social*.
+Unterstütze die Nachhaltigkeit  der Software und deren Netzwerke gerne mit einer Einmal- oder Dauerspende:
+
+<a href="https://busfaktor.org/de/spenden" target="_blank">
+  <Button class="md-button">
+    Hier spenden
+  </Button>
+</a>
```
Comment (Elweyn) [inline docs/de/contribute/README.md:35]: I would set this as last point.

File: docs/de/features/README.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+#     link: https://stage.ocelot.social/
+---
+*Ocelot.social* bietet eine wachsende Vielzahl elementarer und weiterführender Netzwerkfunktionen,
+die durch Spenden der Community erweitert werden können:
+
+<a href="https://busfaktor.org/de/spenden" target="_blank">
+  <Button class="md-button">
+    Hier spenden
+  </Button>
+</a>
```
Comment (Elweyn) [inline docs/de/features/README.md:19]: Would write also this at the end

File: docs/.vuepress/config/theme.js [hunk 4]
```diff
@@ -23,72 +23,106 @@ export default hopeTheme({
     '/de/': {
       lang: 'de-DE',
       title: 'Ocelot.Social',
-      copyright: '&copy; von <a href="https://busfaktor.org" target="_blank">busFaktor() e.V.</a> & Authoren | <a href="/de/impressum/">Impressum</a>',
+      description: 'Hallo',
```
Comment (Tirokk) [inline docs/.vuepress/config/theme.js:26]: This seems wrong

Comment (Tirokk) [reply docs/.vuepress/config/theme.js]: Done

File: docs/de/contribute/README.md (hunk 2 above)
Comment (Tirokk) [reply docs/de/contribute/README.md]: Lets talk please.

File: docs/de/features/README.md (hunk 3 above)
Comment (Tirokk) [reply docs/de/features/README.md]: Lets talk please

File: docs/de/demos/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+title: Demos
+sidebar: false
+tagLine: 
+# actions:
+#   - text: Demo
+#     link: https://stage.ocelot.social/
+---
+## XXX
+
+XXX
```
Comment (mahula) [inline docs/de/demos/README.md:14]: As long as we do not have content here, this page should not be reachable.

File: docs/de/contribute/README.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+Der Verein [busFaktor() e.V.](https://busfaktor.org/de/) koordiniert und fördert die Programmierung von *ocelot.social*.
+Unterstütze die Nachhaltigkeit  der Software und deren Netzwerke gerne mit einer Einmal- oder Dauerspende:
+
+<a href="https://busfaktor.org/de/spenden" target="_blank">
+  <Button class="donate-button">
+    Hier spenden
+  </Button>
+</a>
+
+### Unterstütze dein Netzwerk
```
Comment (mahula) [inline docs/de/contribute/README.md:37]: The purspose of this section is unclear.
An introduction that picks up the user from already being part of an ocelot.social-based network or operating their own network would be helpful.

File: docs/de/contribute/README.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+### Entwickle und unterstütze die Software
+
+#### Mach die *ocelot.social*-Software bekannt
+
+*›Spread the word!‹*
+
+Ob Verein, Bewegung oder Institution – *ocelot.social* kann als soziales Netzwerk für viele Arten von Gruppen und Organisationen eingesetzt werden.
+
+- Veröffentliche den Link zu *ocelot.social* auf deiner Website, deinem Blog oder in sozialen Netzwerken.
```
Comment (mahula) [inline docs/de/contribute/README.md:81]: -Veröffentliche den Link zu *[ocelot.social](https://ocelot.social/)* auf deiner Website, deinem Blog oder in sozialen Netzwerken.

File: docs/de/contribute/README.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+- Gib der Software *ocelot.social* auf der Vergleichsplattform [alternative.to](https://alternativeto.net/software/ocelot-social/about/) ein Like.
+- Gib der Software auf der Entwicklerplattform [GitHub](https://github.com/Ocelot-Social-Community/Ocelot-Social) einen Stern.
+
+#### Verbessere das Netzwerk
+
+- Hast du einen Fehler im Netzwerk gefunden, dann melde ihn unseren Entwicklern (→ [Anleitung im Wiki](https://github.com/Ocelot-Social-Community/Ocelot-Social/wiki/de:FAQ#wie-kann-ich-einen-fehler-des-netzwerks-melden)).
```
Comment (mahula) [inline docs/de/contribute/README.md:87]: - this sentence should clearly and simply refer to the [issue section of the Ocelot social repository](https://github.com/Ocelot-Social-Community/Ocelot-Social/issues).
- the provided e-mail address in the [wiki link](https://github.com/Ocelot-Social-Community/Ocelot-Social/wiki/de:FAQ#wie-kann-ich-einen-fehler-des-netzwerks-melden) does not exist or work (might be an issue for the [Ocelot-Social repo](https://github.com/Ocelot-Social-Community/Ocelot-Social)).

File: docs/de/contribute/README.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+- Hilf mit, die Kommunikationskanäle der Software *ocelot.social* und unserer Entwicklergemeinschaft [busFaktor() e.V.](https://busfaktor.org/de/) zu pflegen und vielfältig zu beleben.
+
+#### Sprichst du eine Fremdsprache?
+
+Das Netzwerk ist in [mehreren Sprachen](/de/features/) verfügbar.
+Möchtest du mithelfen die Benutzeroberfläche oder [Benutzeranleitung im Wiki](https://github.com/Ocelot-Social-Community/Ocelot-Social/wiki/de:Wiki:Editor's-Guide) zu übersetzen, zu vervollständigen oder zu verbessern, dann kontaktiere uns gerne.
+
+#### Möchtest du zum Design beitragen?
+
+Kontaktiere uns, um das Design und die Bedienbarkeit des Netzwerks zu verbessern.
```
Comment (mahula) [inline docs/de/contribute/README.md:106]: [Kontaktiere uns](/de/contact/) gern, um das Design und die Bedienbarkeit des Netzwerks zu verbessern.

File: docs/de/contribute/README.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+
+#### Organisierst du gerne?
+
+Sprich uns an, wenn du Lust hast, …
```
Comment (mahula) [inline docs/de/contribute/README.md:110]: [Sprich uns an](/de/contact/), wenn du Lust hast, …

File: docs/de/about/README.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+# layout: BlogHome
+icon: home
+title: Über
+sidebar: false
+tagLine: 
+# actions:
+#   - text: Demo
+#     link: https://stage.ocelot.social/
+---
+## XXX
```
Comment (mahula) [inline docs/de/about/README.md:12]: Same here:

- As loang as we do not have content here, [this page](/de/contact/) should not be available

**Review by mahula — CHANGES_REQUESTED** [review]
Nice approach.
Thank you big time @sushidave and @Tirokk  for the time and effort you put in making this website happen.

File: docs/de/contact/README.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+Haferstr. 5c  
+86179 Augsburg
+
+Telefon: +49 173 622 60 91  
+Discord: <https://discord.gg/2b55aHVgpM>
+
+## Ocelot.Social-Projekt
+
+Telefon: +49 173 622 60 91  
+E-Mail: <hello@ocelot.social>
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/de/contact/README.md:25]: I would prefer to keep this e-mail, but lets talk.

I wrote to Martin to check and fix the e-mail.

File: docs/.vuepress/config.js [hunk 13]
```diff
@@ -1,16 +1,25 @@
...
   plugins,
   bundler: viteBundler(),
   base: process.env.VUEPRESS_BASE ? `/${process.env.VUEPRESS_BASE}/` : '/',
+  alias: {
+    '@theme': path.resolve(
+      __dirname,
+      './theme',
+    ),
+  },
   locales: {
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/.vuepress/config.js:23]: We thought about it.

Can you do without a lot changes?
Talk?

File: docs/en/imprint/README.md [hunk 14]
```diff
@@ -12,18 +12,20 @@ tagLine:
 ## Responsible for the Operation of this Website
 
 busFaktor() e.V.  
-Wolfgang Huß, 2nd Chairman
-
+Wolfgang Huß, 2nd Chairman  
 Haferstr. 5c  
-86179 Augsburg  
+86179 Augsburg
+
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/en/imprint/README.md:18]: Cool catch. Done.

File: docs/de/contribute/README.md (hunk 8 above)
Comment (Tirokk) [reply docs/de/contribute/README.md:91]: - lets talk about the sentence
- not working e-mail address: see above

File: docs/de/contribute/README.md (hunk 9 above)
Comment (Tirokk) [reply docs/de/contribute/README.md]: Cool catch! Done

File: docs/en/imprint/README.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/en/imprint/README.md:18]: In `contact` I fixed it as well.

File: docs/de/contribute/README.md (hunk 10 above)
Comment (Tirokk) [reply docs/de/contribute/README.md:114]: Done

File: docs/de/about/README.md (hunk 11 above)
Comment (Tirokk) [reply docs/de/about/README.md:12]: See above

File: docs/de/privacy-policy/README.md [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+
+### Webseite
+
+Bei der Nutzung unserer Webseite werden verschiedene Arten von personenbezogen Daten unterschiedlich verarbeitet. Im Folgenden informieren wir Dich darüber, welche Verarbeitungen es gibt, welche personenbezogenen Daten jeweils betroffen sind, was der Zweck der jeweiligen Verarbeitung ist, wie lange die Daten jeweils gespeichert werden und noch einiges mehr.
+
+#### Webseitenzugriff
+
+**Zweck:** Wir stellen unsere Webseite öffentlich zur Verfügung, damit sich jeder über unseren Verein, seine Tätigkeiten und Angebote informieren kann.
+
+**Personenbezogene Daten:** Übertragen werden u.a., je nach verwendetem Browser und dessen Einstellung: Datum und Uhrzeit des Zugriffes, ggf. Quelle/Verweis, von welcher Seite Du auf unsere jeweilige gelangst, verwendeter Browser, verwendetes Betriebssystem, Sprachinformation, Größe des Browserfensters, verwendete IP-Adresse.
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/de/privacy-policy/README.md:47]: Lets talk

File: docs/de/news/README.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
+---
+home: false
+layout: BlogHome
+# icon: home
+title: Neuigkeiten
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/de/news/README.md:5]: Lets talk

File: docs/de/get-started/README.md [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+| `moderator@example.org` | 1234     | Moderator       |
+| `admin@example.org`     | 1234     | Administrator   |
+
+## Hosting auf einem Server
+
+### Selber hosten
+
+*Ocelot.social* kannst du selber hosten.
+Dafür benötigst du umfangreiche Fachkenntnis in der IT.  
+Bei Fragen helfen wir dir vom [busFaktor() e.V.](<https://busfaktor.org/de/>) gerne weiter.
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/de/get-started/README.md:30]: Toppi. 👍🏼
Done all over the place.

File: docs/de/privacy-policy/README.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/de/privacy-policy/README.md:47]: Later

File: docs/de/demos/README.md (hunk 5 above)
Comment (Tirokk) [reply docs/de/demos/README.md:14]: Done

File: docs/de/contribute/README.md (hunk 6 above)
Comment (Tirokk) [reply docs/de/contribute/README.md:39]: did a notice in the EPIC #49

File: docs/de/about/README.md (hunk 11 above)
Comment (Tirokk) [reply docs/de/about/README.md:12]: Done

File: docs/.vuepress/config.js (hunk 13 above)
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/.vuepress/config.js:23]: @mahula likes to care for ist

**Comment by Tirokk** [comment]
If you could check for the failing linter as well @mahula 
https://github.com/Ocelot-Social-Community/ocelot.social/actions/runs/9175179464/job/25227595838?pr=55

File: docs/.vuepress/config.js (hunk 13 above)
[the remark this answers is not in the document]
Comment (mahula) [reply docs/.vuepress/config.js:21]: Done

**Review by mahula — APPROVED** [review]

**Review by Mogge — APPROVED** [review]

**Pull request merged**